src/stdlib/lv_string.h

```diff
@@ -114,6 +123,15 @@ char * lv_strdup(const char * src);
  */
 char * lv_strdup(const char * src);
 
+/**
+ * @brief Copies the string pointed to by src, including the terminating null character,
+ *        to the end of the string pointed to by dst.
+ * @param dst Pointer to the destination string where the content is to be appended.
+ * @param src Pointer to the source of data to be copied.
+ * @return A pointer to the destination string, which is dst.
+ */
+char * lv_strcat(char * dst, const char * src);
```

Comment:
Should we also add version of lv_strncat?

Reply:
It would make sense to replace all possible calls to lv_strcat with lv_strncat. IMO also all calls to lv_strcpy should be changed to lv_strncpy. 
But i think that should be done in a new PR.

Reply:
Having only `lv_strncat` makes sense. 

I agree to to spare `lv_strncpy` for an other PR.

Reply:
`lv_strncat` PR: #5927